.buildkite/pipeline.yml

```diff
@@ -139,97 +131,61 @@ steps:
+        artifact_paths: "sphere_aquaplanet_rhoe_equilmoist_allsky_gw/*"
 
       - label: ":computer: aquaplanet (ρe_tot) equilmoist with EDMF"
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --moist dry --anelastic_dycore false --job_id sphere_baroclinic_wave_rhoe_equilmoist_mo_edmf --vert_diff true --surface_scheme bulk --dt 1secs --t_end 20secs --FLOAT_TYPE Float64 --turbconv edmf --turbconv_case Soares --config sphere --precip_model 0M --z_stretch false --post_process  false --apply_limiter false --dt_save_to_sol 10secs --dt_save_to_disk 10secs"
         artifact_paths: "sphere_baroclinic_wave_rhoe_equilmoist_mo_edmf/*"
         agents:
           slurm_mem: 20GB
-
-  - group: "Sphere Examples (Gravity wave)"
-    steps:
-
-      - label: ":computer: baroclinic wave with orographic gravity wave"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --orographic_gravity_wave true --job_id sphere_baroclinic_wave_ogw --t_end 1days --regression_test true"
-        artifact_paths: "sphere_baroclinic_wave_ogw"
-
-      - label: ":computer: held suarez (ρe) equilmoist hightop gw parameterization"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --z_max 45e3 --z_elem 50 --dz_top 3e3 --vert_diff true --surface_scheme bulk --moist equil --forcing held_suarez --precip_model 0M --non_orographic_gravity_wave true --job_id sphere_held_suarez_rhoe_equilmoist_hightop_nogw --dt 450secs --t_end 1days --regression_test true"
-        artifact_paths: "sphere_held_suarez_rhoe_equilmoist_hightop_nogw/*"
       
   - group: "Sphere Examples (Topography)"
     steps:
 
-      - label: ":computer: held suarez (ρe_tot) topography (dcmip)"
+      - label: ":computer: held suarez (ρe) topography (dcmip)"
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --forcing held_suarez --job_id sphere_held_suarez_rhoe_topography_dcmip --t_end 6days --topography DCMIP200 --dt 200secs --regression_test false --rayleigh_sponge true --viscous_sponge true"
         artifact_paths: "sphere_held_suarez_rhoe_dcmip_topography_dcmip/*"
 
-      - label: ":computer: baroclinic wave (ρe_tot) topography (dcmip)"
+      - label: ":computer: baroclinic wave (ρe) topography (dcmip)"
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --job_id sphere_baroclinic_wave_rhoe_topography_dcmip --topography DCMIP200 --dt 200secs --regression_test false --rayleigh_sponge true --viscous_sponge true"
         artifact_paths: "sphere_baroclinic_wave_rhoe_topography_dcmip/*"
 
```

Comment:
```suggestion
```
We can remove this case since the next item in the list checks a version of this without the viscous sponge active.